packages/alert/src/AlertBase.tsx

```diff
@@ -49,26 +50,41 @@ export const AlertWrapper = styled.div<BaseAlertProps>`
 `
 
 export const AlertMessages = styled.div<Partial<AlertProps>>`
-  margin-left: ${rem(0.8)};
+  margin-left: ${rem(1)};
   display: flex;
   flex-direction: column;
   justify-content: center;
 `
 
 export const AlertTitle = styled.span<Partial<AlertProps>>`
-  font: ${weight.bold} ${p3.fontSize} ${family};
-  line-height: ${p3.lineHeight};
+  font: ${weight.bold} ${p2.fontSize} ${family};
+  line-height: ${p2.lineHeight};
+
+  @media (max-width: ${breakpoints.sm}px) {
+    font: ${weight.bold} ${p3.fontSize} ${family};
+    line-height: ${p3.lineHeight};
+  }
 `
 
 export const AlertDescription = styled.span<Partial<AlertProps>>`
-  font: ${weight.normal} ${p4.fontSize} ${family};
-  line-height: ${p4.lineHeight};
+  font: ${weight.normal} ${p3.fontSize} ${family};
+  line-height: ${p3.lineHeight};
   vertical-align: middle;
+
+  @media (max-width: ${breakpoints.sm}px) {
+    font: ${weight.normal} ${p4.fontSize} ${family};
+    line-height: ${p4.lineHeight};
+  }
 `
```

Comment:
You could do something like this instead:
https://github.com/little-spoon-dev/little-spoon-frontend/blob/8b256db18696a31c36e2fe86b18a36f08776770d/app/pages/Account/MyBox/Content/ProductsGroups/ProductsGroup/styled.js#L81-L91